Description: Correct soname version of library
 Upstream forgot to update the soname in the new release.  Include
 the change here.
Author: Petter Reinholdtsen <pere@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: https://github.com/ptrkrysik/gr-gsm/commit/d30d0c23e35b818cc1dc573dc00354676649fd31
Forwarded: not-needed
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2018-08-11

--- gr-gsm-0.42.1.orig/CMakeLists.txt
+++ gr-gsm-0.42.1/CMakeLists.txt
@@ -60,8 +60,8 @@ endif()
 
 # Set the version information here
 set(VERSION_INFO_MAJOR_VERSION 0)
-set(VERSION_INFO_API_COMPAT    41)
-set(VERSION_INFO_MINOR_VERSION 4)
+set(VERSION_INFO_API_COMPAT    42)
+set(VERSION_INFO_MINOR_VERSION 1)
 set(VERSION_INFO_MAINT_VERSION git)
 include(GrVersion) #setup version info
 
